src/components/SiteHeader.js

```diff
@@ -73,24 +85,49 @@ const SiteHeader = () => {
                 url: "/data/explorer",
                 icon: <BarChartOutlined />,
                 text: "Explorer",
-                disabled: !isAuthenticated,
+                disabled: !canQueryData,
                 key: "explorer",
             },
-            // TODO: Only show if admin / can data manage anything
             {
-                key: "data-manager",
-                url: "/data/manager",
-                icon: <FolderOpenOutlined />,
-                text: "Data Manager",
-                disabled: !isAuthenticated,
-            },
-            {
-                key: "services",
-                url: "/services",
-                icon: <DashboardOutlined />,
-                text: "Services",
-                disabled: !isAuthenticated,
+                url: "/genomes",
+                icon: <FileTextOutlined />,
+                text: "Reference Genomes",
+                key: "genomes",
             },
+            ...(managerPermissions.canManageAnything ? [
+                {
+                    key: "data-manager",
+                    url: "/data/manager",
+                    icon: <FolderOpenOutlined />,
+                    text: "Data Manager",
+                    disabled: !isAuthenticated || !managerPermissions.canManageAnything,
```

Comment:
not sure `disabled` is doing anything here:
- managerPermissions.canManageAnything is always true if we made it to this code, so right side of disjunction is always false.
- I'm guessing left side is always false for the same reasons